pkgs/tools/networking/croc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "croc";
+  version = "6.1.1";
+
+  goPackagePath = "github.com/schollz/croc";
+
+  src = fetchFromGitHub {
+    owner = "schollz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "08gkwllk3m5hpkr1iwabvs739rvl6rzdnra2v040dzdj6zgyd12r";
+  };
+
+  modSha256 = "026m3hc2imna7bf4jpqm7yq6mr4l5is2crsx1vxdpr4h0n6z0v3i";
+  subPackages = [ "." ];
+
+  meta = with stdenv.lib; {
+    description     = "Easily and securely send things from one computer to another. Written in Go";
+    homepage        =  https://schollz.com/software/croc6;
+    license         = licenses.mit;
+    maintainers     = with maintainers; [ hugoreeves ];
+    platforms       = platforms.unix;
+
+    longDescription = ''
+    Croc is a tool that allows any two computers to simply and securely transfer files and folders. AFAIK, croc is the only CLI file-transfer tool does all of the following:
```

Comment:
You can indent this since `''` automatically strips `\n`. These two sentences on their own newlines would be good too :+1: